R/plotPrevalence.R

```diff
@@ -209,12 +220,19 @@ setMethod("plotPrevalentAbundance", signature = c(x = "SummarizedExperiment"),
              size_by = NULL,
              shape_by = NULL,
              label = NULL,
+             facet_by = FALSE,
```

Comment:
It would be easier to follow the other variables by setting the default to `NULL`

Reply:
yes, true

--> NULL